CHANGELOG.rst

```diff
@@ -1,5 +1,10 @@
-lib_sw_pll library change log
-=============================
+lib_sw_pll change log
+=====================
+
+UNRELEASED
```

Comment:
You may find this breaks the lib checks. It may want a placeholder actual version number eg,. 2.20

Reply:
Also, settings.yml will have the old version still - so you could end up with docs and code which are different from the official version.

Reply:
The lib checks pass with the updated commit in infr_apps.